tests/browser/test/basic_playground_test.js

```diff
@@ -95,42 +90,126 @@ suite('Right Clicking on Blocks', function (done) {
     await contextMenuSelect(browser, block, 'Collapse Block');
     await contextMenuSelect(browser, block, 'Expand Block');
```

Comment:
Would it be worth verifying it is actually collapsed before expanding?  Or is that just duplicating the previous test?  Could you collapse the block via API for this test?  Otherwise maybe combine the tests since they seem to be unavoidably entwined.

And same below for disable/enable.

Reply:
I think collapsing via API makes sense! although honestly I'm not really sure how to write a good integration test :P

Reply:
Reverted after talking with Rachel! She changed these to all chain.